src/documentation/domains/project/controller/schemas.docs.ts

```diff
@@ -88,4 +88,22 @@
  *         name: "Backlog"
  *         type: "BACKLOG"
  *         color: "#bec2c8"
+ *
+ *     ProjectBasicDTO:
```

Comment:
Consider using `BasicProjectDataDTO` because the structure is pretty similar, so you don't repeat code. Or try changing dto's name for something else.

Reply:
I have changed Classes names to avoid misunderstandings